pkg/backend/crypto/gpg/cli/import_windows_test.go

```diff
@@ -0,0 +1,21 @@
+package cli
```

Comment:
import_linux_test.go should also test OSX right? so maybe import_unix_test.go would be better.

Reply:
That would not work ([see here](https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool)) The convention is `file_$GOOS[_test].go`. I found a few files in the package that use`foo_others[_test].go` with a build constraint `// +build !windows`.